torchci/components/GroupJobConclusion.tsx

```diff
@@ -34,6 +34,37 @@ export enum GroupedJobStatus {
   Pending = "pending",
 }
 
+type RepoViableStrictBlockingJobsMap = {
+  [key: string]: RegExp[];
+};
+
+// TODO: Move this to a config file
```

Comment:
https://github.com/pytorch/pytorch/blob/main/.github/pytorch-probot.yml seems to be a a natural place for this, but agree that we could do it later